.github/workflows/ansible-test.yml

```diff
@@ -1,25 +1,67 @@
 name: CI
-
 on: push
 
 jobs:
-  build:
+  sanity:
     runs-on: ubuntu-latest
     steps:
+
+      - name: Check out code
+        uses: actions/checkout@v1
+        with:
+          path: ansible_collections/gundalow_collections/grafana
+
       - name: Set up Python 3.6
         uses: actions/setup-python@v1
         with:
           python-version: 3.6
+
+      - name: Install Ansible 2.9.1
+        run: pip install ansible==2.9.1 --disable-pip-version-check
```

Comment:
why does the ansible version is fixed ? maybe we can just use the "latest" ansible version ? If we fix the version in the tests we will have to update it each time a new release is out.